site/docs/11-input/10.3-pointer.mdx

```diff
@@ -1,162 +1,196 @@
+You also have the option to handle _all_ pointer events in the browser window by using [[Input.PointerScope.Document|PointerScope.Document]], like this:
 
-```js
-engine.input.pointers.primary.on('down', function (pe) {
-  if (pe.pointerType === ex.Input.PointerType.Mouse) {
-    ex.Logger.getInstance().info('Mouse event:', pe);
-  } else if (pe.pointerType === ex.Input.PointerType.Touch) {
-    ex.Logger.getInstance().info('Touch event:', pe);
-  }
+```ts twoslash
+// @noErrors
+/// <reference path="../src/engine/excalibur.d.ts" />
+// ---cut---
+const game = new ex.Engine({
+  pointerScope: ex.Input.PointerScope.D
+//                                     ^|
 });
 ```
 
-### Multiple Pointers (Multi-Touch)
+This is useful for handling complex input and having control over every interaction.
 
-When there is more than one pointer detected on the screen,
-this is considered multi-touch. For example, pressing one finger,
-then another, will create two pointers. If you lift a finger,
-the first one remains and the second one disappears.
+:::note When to Use Document Scope
 
-You can handle multi-touch by subscribing to however many pointers
-you would like to support. If a pointer doesn't yet exist, it will
-be created. You do not need to check if a pointer exists. If it does
-exist, it will propagate events, otherwise it will remain idle.
+One real-world example is dragging and gestures. Sometimes a player will drag their
+finger outside your game and then into it, expecting it to work. If [[Input.PointerScope|PointerScope]]
+is set to [[Input.PointerScope.Canvas|Canvas]] this will not work. If it is set to
+[[Input.PointerScope.Document|Document]], it will.
 
-Excalibur does not impose a limit to the amount of pointers you can
-subscribe to, so by all means, support all 10 fingers.
+:::
 
-_Note:_ There is no way to identify touches after they happen; you can only
-know that there are _n_ touches on the screen at once.
+:::info What about HTML elements?
+`PointerScope.Canvas` does not affect handling native HTML events like `click`, `mouseover`, `mouseout`, etc. on any HTML-based UI. It only applies to pointer events that are targeted to the HTML `<canvas>` element Excalibur uses to render your game.
 
-```js
-function paint(color) {
-  // create a handler for the event
-  return function (pe) {
-    if (pe.pointerType === ex.Input.PointerType.Touch) {
-      engine.canvas.fillStyle = color;
-      engine.canvas.fillRect(pe.x, pe.y, 5, 5);
-    }
-  };
-}
-engine.input.pointers.at(0).on('move', paint('blue')); // 1st finger
-engine.input.pointers.at(1).on('move', paint('red')); // 2nd finger
-engine.input.pointers.at(2).on('move', paint('green')); // 3rd finger
-```
+However, `PointerScope.Document` will handle all pointer events in the browser window, including those targeted to HTML elements. If you overlay any HTML elements on top of your game, be aware that pointer events will also be handled by Excalibur.
+:::
```

Comment:
@eonarheim need confirmation on this, I think this is true?

Reply:
I believe that is true, the pointer event receiver is listening to the Document for any events in that case. So you need to be careful.